src/EditorFeatures/Core/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -253,5 +254,25 @@ internal interface IInlineRenameInfo
 /// </summary>
 internal interface IEditorInlineRenameService : ILanguageService
 {
+    /// <summary>
+    /// Returns <see cref="IInlineRenameInfo"/> necessary to establish the inline rename session.
+    /// </summary>
     Task<IInlineRenameInfo> GetRenameInfoAsync(Document document, int position, CancellationToken cancellationToken);
+
+    /// <summary>
+    /// Returns whether this language service can return optional context from <see cref="GetRenameContextAsync(IInlineRenameInfo, IInlineRenameLocationSet, CancellationToken)"/>.
+    /// </summary>
+    bool IsRenameContextSupported { get; }
```

Comment:
can remove this and just have GetRenameContextAsync return null.

Reply:
Yeah feel free to refactor this based on Cyrus's comment.